groups/committee-steering/groups.yaml

```diff
@@ -1,4 +1,35 @@
 groups:
+  - email-id: steering-emeritus@kubernetes.io
+    name: steering-emeritus
+    description: |-
+      Private list for current and emeritus members of the Kubernetes Steering Committee
+    settings:
+      ReconcileMembers: "true"
+    owners:
+      - bburns@microsoft.com
+      - briangrant@google.com
+      - cblecker@gmail.com
+      - davanum@gmail.com
+      - decarr@redhat.com
+      - jbeda@gcp.vmware.com
```

Comment:
The `gcp.vmware.com` email no longer works afaik.

@jbeda which email would you prefer here?

Reply:
> The `gcp.vmware.com` email no longer works afaik.

Yep, this was a temp address for the VMware emails that was only in use during the email account migrations a while back.